test/old_tests/UnitTests/constexpr.cpp

```diff
@@ -38,7 +38,7 @@ TEST_CASE("constexpr")
 
     REQUIRE(winrt::name_of<winrt_container>() == midl_container::z_get_rc_name_impl());
 
-    constexpr auto name = winrt::name_of<winrt::guid>();
+    auto name = winrt::name_of<winrt::guid>();
```

Comment:
The change makes this not really a constexpr test any more. Shouldn't this test still be using `name_v` and comparing it to `u8"Guid"`?

Reply:
Yeah I should just remove this test.